calculation/gmhazard_calc/gmhazard_calc/dbs/IMDB.py

```diff
@@ -341,7 +341,7 @@ def im_data(
         im: IM or list[IM], optional
             IM(s) of interest
             if this is unspecified then it is equivalent to setting all IMs
-        inter_intra: bool
+        incl_inter_intra: bool
             Boolean flag to determine to either extract normal mu and sigma or mu, sigma_inter and sigma_intra
```

Comment:
Not yours, but still worth fixing. Should be 
`..... extract mu and total standard deviation or mu, between-event and within-event standard deviation`